.github/workflows/labeler.yml

```
@@ -0,0 +1,14 @@
+name: pr-labeler
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v3
+      with:
+        repo-token: "${{ secrets.RIOT_CI_ACCESS_TOKEN }}"
```

I think you need to remove the `"`s.

```suggestion
        repo-token: ${{ secrets.RIOT_CI_ACCESS_TOKEN }}
```

---

Are you sure this token exists for the RIOT repo ? The error seems to say that it's empty